pages/viewBattles.php

```diff
@@ -106,5 +106,115 @@ function viewBattles() {
     }
 
     require 'templates/viewBattles.php';
+
+    /* Begin User Battle History */
+    if ($player->rank_num > 1 && $player->forbidden_seal->max_battle_history_view > 0) {
```

Comment:
Should we make this like, a tab players can switch between? Like

"Live Battles" | "Your Battle History"

Reply:
I think that's a great improvement, will add to the list.